File: Java/Program/RAM.java
public class RAM extends Component {
    private int size;
    private String ddr;
    private int frequency;

    public RAM() {}

    public RAM(String brand, String model, int size, String ddr, int frequency) {
        super(brand, model);
        this.size = size;
        this.ddr = ddr;
        this.frequency = frequency;
    }

    public void setSize(int size) { this.size = size; }
    public void setDdr(String ddr) { this.ddr = ddr; }
    public void setFrequency(int frequency) { this.frequency = frequency; }

    public int getSize() { return size; }
    public String getDdr() { return ddr; }
    public int getFrequency() { return frequency; }
}

File: Python/Program/RAM.py
from Component import Component

class RAM(Component):
    def __init__(self, brand="", model="", size=0, ddr="", frequency=0.0):
        super().__init__(brand, model)
        self.size = size
        self.ddr = ddr
        self.frequency = frequency

    def set_size(self, size):
        self.size = size

    def set_ddr(self, ddr):
        self.ddr = ddr

    def set_frequency(self, frequency):
        self.frequency = frequency

    def get_size(self):
        return self.size

    def get_ddr(self):
        return self.ddr

    def get_frequency(self):
        return self.frequency
